packages/page-staking/src/useSortedTargets.ts

```diff
@@ -64,18 +64,20 @@ function sortValidators (list: ValidatorInfo[]): ValidatorInfo[] {
 }
 
 function extractSingle (allAccounts: string[], amount: BN = baseBalance(), { info }: DeriveStakingElected | DeriveStakingWaiting, favorites: string[], perValidatorReward: BN, isElected: boolean): [ValidatorInfo[], string[], BN, BN] {
-  const nominators: string[] = [];
+  const defaultExposure = {
```

Comment:
Erm, was not meant to be includes here, finger error - extracted this specific on in https://github.com/polkadot-js/apps/pull/3529 (innocent, but really doesn't belong here, part of a larger PR elsewhere)